docs/GUIDE.md

```diff
@@ -0,0 +1,142 @@
+- Use Circe Akka Serializer but without enabling Compiler Plugins
+- Use Compiler Plugins but not Circe Akka Serializer (if you need to stick to your current Serializer)
+
+To use ASH (whole or a selected part), you need to first add Akka Serialization Helper in the `project/plugins.sbt` file and enable it in `build.sbt` &mdash; this is the standard way of using the toolbox. Just follow short instructions from [install part of README](https://github.com/VirtusLab/akka-serialization-helper#install).
+
+Alternatively &mdash; if you want to use only Circe Akka Serializer (without enabling compiler plugins) &mdash; you can add it as a standard library dependency (find new available versions under [releases](https://github.com/VirtusLab/akka-serialization-helper/releases)):
+```sbt
+libraryDependencies += "org.virtuslab.ash" %% "circe-akka-serializer" % "Version"
+```
+
+## How to use:
+1. [Circe Akka Serializer](#circe-akka-serializer)
+2. [Annotations](#annotations)
+3. [Serializability Checker Compiler Plugin](#serializability-checker-compiler-plugin)
+4. [Codec Registration Checker Compiler Plugin](#codec-registration-checker-compiler-plugin)
+5. [Dump Persistence Schema Compiler Plugin](#dump-persistence-schema-compiler-plugin)
+6. [ashDumpPersistenceSchema sbt task](#ashdumppersistenceschema-sbt-task)
+
+### Circe Akka Serializer
+[CirceAkkaSerializer](../circe-akka-serializer/src/main/scala/org/virtuslab/ash/circe/CirceAkkaSerializer.scala) is the main abstract class that you have to extend to use as the Serializer in your application. Before extending it, read Scaladoc comments available in following files:
+- [CirceAkkaSerializer.scala](../circe-akka-serializer/src/main/scala/org/virtuslab/ash/circe/CirceAkkaSerializer.scala) (the most important starting point / main abstraction)
+- [CirceTraitCodec.scala](../circe-akka-serializer/src/main/scala/org/virtuslab/ash/circe/CirceTraitCodec.scala) (a trait that is extended by CirceAkkaSerializer and has some important `val`s that should be overridden in your code)
+- [Register.scala](../circe-akka-serializer/src/main/scala/org/virtuslab/ash/circe/Register.scala) (helper object with `apply` method that needs to be used in order to register Codecs for serializable type)
+- [Registration.scala](../circe-akka-serializer/src/main/scala/org/virtuslab/ash/circe/Registration.scala) (`Register.apply` method returns an instance of type Registration. Registration instances are later used as registered codecs)
+
+Scaladocs from these files and a quick lecture of [Akka serialization guide](https://doc.akka.io/docs/akka/current/serialization.html) should be enough to use Circe Akka Serializer properly. Apart from implementing the Circe Akka Serializer in scala code, remember to configure your Serializer in Akka-specific configuration file (.conf file). Two mandatory configurations are `akka.actor.serializers` and `akka.actor.serialization-bindings` (example below):
```

Comment:
`lecture` doesn't seem to have the sense of _lektura_, i.e. _the process of reading_ in English.

```suggestion
Scaladocs from these files and a quick read of [Akka serialization guide](https://doc.akka.io/docs/akka/current/serialization.html) should be enough to use Circe Akka Serializer properly. Apart from implementing the Circe Akka Serializer in scala code, remember to configure your Serializer in Akka-specific configuration file (.conf file). Two mandatory configurations are `akka.actor.serializers` and `akka.actor.serialization-bindings` (example below):
```